src/components/InternalDropdown/index.js

```diff
@@ -207,10 +224,17 @@ const InternalDropdown = forwardRef((props, reference) => {
         return handleChange(rest);
     };
 
+    const handleTabKeyPressed = event => {
+        if (showCheckbox && event.target.tagName !== 'BUTTON') {
+            event.stopPropagation();
```

Comment:
we must avoid doing `stopPropagation` take a look at what we did in Tree component (src/components/Tree/helpers/shouldSelectNode.js), instead of calling stopPropagation we did some kind of filter
What happens is that if someone use this component and in its implementation, he uses a global listener then it will not get this event, we must avoid this behavior.
I know we use it in other place and we need to fix them, but for now what we can do is no doing it anymore

Reply:
Done